tests/storage/CMakeLists.txt

```diff
@@ -5,8 +5,18 @@ find_package(Zephyr REQUIRED HINTS $ENV{ZEPHYR_BASE})
 
 project(storage_test)
 
+# ######################################################################################################################
+# Config
+# ######################################################################################################################
+
+set(AOS_CORE_CONFIG aoscoreconfig.hpp)
```

Comment:
not sure this definition is required as it is already defined in the main cmake file

Reply:
Tests are independent applications. It is the top layer for each test.